contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,22 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually. This is possible either via the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` or using Git in the terminal:
+```bash
+git checkout main
+git remote add upstream git@github.com:tldr-pages/tldr.git # only run this if you don't already have the upstream remote for https://github.com/tldr-pages/tldr.git
```

Comment:
Though it's probably better to use SSH, not everyone will have SSH keys on GitHub, and HTTPS will always work.
```suggestion
git remote add upstream https://tldr-pages/tldr.git # only run this if you don't already have the upstream remote for https://github.com/tldr-pages/tldr.git
```